### DISCUSSION TITLE
Creating a shareable link - similar to a google drive or notion link

### DISCUSSION THREAD

**Discussion by ethanfox** [supabase/supabase#13352] [Questions]
I am creating an app for interior designers and architects to share material specifications with their clients. Basically, after the client has agreed to the design direction the designer needs to create a list that the client to refer to when ordering the different materials. 

It looks like this:
<img width="725" alt="Screenshot 2023-03-31 at 12 58 06 PM" src="https://user-images.githubusercontent.com/22962600/229195292-d6d0432d-9d60-415c-8494-c61c512b926c.png">

How do I make a shareable link that wouldn't require a login to allow the table to be viewable? (Similar to the question you get when sharing a Google doc - The person with this link can... View, Edit, etc) 

Does anyone have any experience with this? Would something like this even involve supabase?

**Comment by GaryAustin1** [comment]
You could use a UUID token as part of the URL parameters.  Then store that token in a column for either the project or with each row that "belongs" to the token.  UUID is not really guessable, but can be shared in a link.
